health/health.d/entropy.conf

```diff
@@ -15,5 +15,8 @@ component: Cryptography
     every: 5m
      warn: $this < (($status >= $WARNING) ? (200) : (100))
     delay: down 1h multiplier 1.5 max 2h
-     info: minimum number of entries in the random numbers pool in the last 5 minutes
+     info: Minimum number of entries in the random numbers pool in the last 5 minutes. \
+           A low number of entries in the random numbers pool. \
+           It may affect the performance of applications that use the pool. \
```

Comment:
Actually, it generally won’t impact performance at all on modern Linux systems, it impacts quality of security for things like TLS connections.

`/dev/random` on all modern Linux systems is identical to `/dev/urandom`, except it blocks on startup until enough entropy has been accumulated to securely seed the underlying PRNG.

Reply:
Not a specialist, I was under the impression, that 
1. a poor source of randomness could result in loss of security.
2. lack of bits in the random pool could result in applications waiting for the randomness stalls until more random stuff is available => affects performance.

> it generally won’t impact performance at all on modern Linux systems

Do you mean 2nd never happens?